invenio_notifications/views/ui.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
```

Comment:
minor: I think we could just get away with a single `views.py` file (since this module will probably not have UI + REST API views.